src/main/java/org/prebid/server/bidder/ix/IxBidder.java

```diff
@@ -222,7 +233,7 @@ private BidderBid toBidderBid(Bid bid, BidRequest bidRequest, BidResponse bidRes
         final boolean bidHasNoSizes = bid.getH() == null || bid.getW() == null;
         final Banner banner = bidRequest.getImp().get(0).getBanner();
 
-        if ((bidHasNoSizes && banner != null) || (extVideo != null)) {
+        if (bidHasNoSizes && banner != null || extVideo != null) {
             final Bid.BidBuilder bidBuilder = bid.toBuilder();
 
             if (bidType == BidType.banner && bidHasNoSizes && banner != null) {
```

Comment:
There is no this code in Go implementation, first of all we need updates for this bidder